src/library/OverflowContainer/OverflowContainer.js

```diff
@@ -82,11 +99,15 @@ export default class OverflowContainer extends Component<Props, State> {
   handleWindowResize = debounce(this.updateScrollable, 500);
 
   render() {
-    const { children, ...restProps } = this.props;
+    const { children, tabIndex, ...restProps } = this.props;
```

Comment:
I am getting occasionally seeing some exceptions thrown from this component.  I think that maybe you just need to guard that `node` exists in `updateScrollable`.

![screen shot 2018-09-20 at 10 43 27 am](https://user-images.githubusercontent.com/202773/45833601-47423a00-bcc2-11e8-95f8-386307a33e7f.png)